source/write/bulk-write.txt

```diff
@@ -20,19 +20,33 @@ Bulk Write Operations
 Overview
 --------
 
-Consider a scenario in which you want to insert a document into a collection,
-update multiple other documents, then delete a document. If you use
-individual methods, each operation requires its own database call. This guide
-shows you how to use bulk write operations to perform multiple write operations
-in a single database call.
+In this guide, you can learn how to use {+driver-short+} to
+perform **bulk operations**. Bulk operations reduce the number
+of calls to the server by performing multiple write operations
+in a single method.
+
+You can use bulk operations to perform multiple write operations on
+a collection. You can also call bulk operations on your client,
+which allows you to perform bulk writes across multiple namespaces.
```

Comment:
I'd probably say, "Both collections and clients have bulk_write methods. Using the latter allows you to perform bulk writes across multiple namespaces."

Reply:
I reworded, let me know what you think!